### ISSUE TITLE
Error message on heroku run due to bad  dyno type could be clearer

### ISSUE THREAD

**Issue by uhoh-itsmaciek** [heroku/heroku-run#11]
If I try to heroku run on a new app with a dyno running, I get an error about Free dyno limits, but no indication on how to fix it:

```console
maciek@mothra:~/code/aux/newts-poc$ heroku run init
Running init on newts-testing... !!!
 ▸    Cannot run more than 1 Free size dynos.
```

It'd be great if this could link to the [dyno type documentation](https://devcenter.heroku.com/articles/dyno-types#setting-dyno-type) or generally give more hints as to how to resolve the issue.

**Comment by dickeyxxx** [comment]
Definitely. And I agree with your comment you made that we should work to have really good, full error messages like we do for multiple remotes.

Trouble is this error comes from the API. We need a good way to handle these errors so that we can show the extra context.

**Comment by dickeyxxx** [comment]
Because the message we should show here would probably contain content that would make sense in the CLI, but not as an API response. Unless maybe the API looked at the user-agent... but eh. I'm not so sure that would be a good idea.

**Comment by dickeyxxx** [comment]
@uhoh-itsmaciek @heroku/api I could wrap this error like I do for others, so this should be all client side work. However, what message could I show here? `heroku ps` `heroku ps:kill`?

**Comment by uhoh-itsmaciek** [comment]
> However, what message could I show here? `heroku ps` then `heroku ps:kill`?

Sorry, I don't follow...

**Comment by dickeyxxx** [comment]
I'm asking what instruction could I show to the user. Something like:

```
Cannot run more than 1 Free size dynos.
To see which dynos are currently running, run `heroku ps`.
If you want to kill a running dyno, run `heroku ps:kill DYNO`.
```

**Comment by uhoh-itsmaciek** [comment]
Oh interesting. I didn't even think of solving the problem that way: since I definitely wanted to keep things running, I was hoping for something like a Dec Center link to tell me about dyno types and how to resize. But reflecting on this, your approach is also reasonable for a different usage.

**Comment by dickeyxxx** [comment]
I think that's good too! Maybe both would be good for this.
